index.html

```diff
@@ -1538,10 +1538,12 @@ <h3>Algorithm</h3>
           and <a>processing mode</a> is `json-ld-1.0`,
           a <a data-link-for="JsonLdErrorCode">keyword redefinition</a> error has
           been detected and processing is aborted.
-          At this point, <var>value</var>
-          MUST be a <a>map</a> with only the entry <code>@container</code>
-          and value <code>@set</code>
-          and optional entry `@protected`.
+          At this point,
+          <var>value</var> MUST be a <a>map</a> with either or both of the following <a>entries</a>:
```

Comment:
```suggestion
          <var>value</var> MUST be a <a>map</a> with only either or both of the following <a>entries</a>:
```
I wouldn't delete "only", because without it, it seems that other keys would be ok (as long as at least one of those is present). Granted, the sentence afterwards exclude any other key, but I find it clearer that way.